test/atomic_enum_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class TestAtomicEnum < MiniTest::Unit::TestCase
```

Comment:
hrm ok if all the other tests are already doing it wrong then leave it like this... :-(